Module tutorial fix (#1676)
PrestaShop/docs: modules/creation/displaying-content-in-front-office.md

    return parent::install() &&
        $this->registerHook('displayLeftColumn') &&
        $this->registerHook('actionFrontControllerSetMedia') &&
        $this->registerHook('displayRightColumn') &&
        Configuration::updateValue('MYMODULE_NAME', 'my module');
}
```

As you can see, we make it so that the module is hooked to the `displayLeftColumn` and `actionFrontControllerSetMedia` hooks. 
In addition to this, we will add code for the `displayRightColumn` hook.

{{% notice note %}}

> Those hooks will be displayed if your theme is running a three column layout. 

Reviewer: ```suggestion
Those hooks will be displayed if your theme uses a three-column layout. 
```

> To enable three column layout, you need to : 

Reviewer: ```suggestion
To enable a three-column layout, you need to: 
```

> - go to back office > Design > Theme & logo > Configure your page layouts > Choose layouts

Reviewer: ```suggestion
- go to the Back office > Design > Theme & logo > Configure your page layouts > Choose layouts
```

> - and enable "Three Columns" layout on pages

Reviewer: ```suggestion
- enable "Three Columns" layout on selected pages
```
